packages/rust/project.bri

```diff
@@ -160,7 +160,7 @@ export function cargoBuild(options: CargoBuildOptions) {
 
     # If the crate has a .cargo/config file, then move it to .cargo/config.toml
     # Cargo prefers the config over config.toml, so we need to rename it
-    # to avoid any conflits. It will still need to be removed from the merged
+    # to avoid any conflicts. It will still need to be removed from the merged
```

Comment:
I really wish I had a good way of spellchecking comments 😅

Reply:
I, personally, used this extension in VSCode: https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker